hazelcast/src/test/resources/log4j.properties

```diff
@@ -29,3 +29,6 @@ log4j.logger.com.hazelcast.spi.hotrestart.cluster=debug
 log4j.logger.com.hazelcast.test.mocknetwork=debug
 
 log4j.rootLogger=info, stdout
+
+# WARNING: This config file is no longer used in Hazelcast tests
```

Comment:
Hmm, maybe they are or were used to easily test all logging frameworks. Then let's keep it as long as we support log4j.